packages/components/src/Popover/Popover.test.tsx

```diff
@@ -47,6 +47,65 @@ describe("Non-composable Popover", () => {
     expect(screen.queryByText(content)).toBeNull();
   });
 
+  describe("preferredPlacement", () => {
+    it("should use bottom placement by default", () => {
+      renderPopover({
+        open: true,
+        children: content,
+      });
+
+      const popover = screen.getByTestId("ATL-Popover-Container");
+      expect(popover).toHaveAttribute("data-popper-placement", "bottom");
```

Comment:
I feel like this is bordering on testing the library's functionality since we're getting `placement` from floating ui, then passing it down and putting those attributes on a div....

though actually I guess the default bottom is unique to our setup. nevermind, I like it.